src/website/src/app/documentation/demos/header/header-types.demo.html

```diff
@@ -125,7 +125,7 @@
                 <clr-dropdown>
                     <button class="nav-text" clrDropdownTrigger>
```

Comment:
I'm wondering if we somehow could make the `aria-label` part of the `clrDropdownTrigger` directive?

Reply:
I'm thinking something like this:


```html
<button 
  class="nav-text" 
  clrDropdownTrigger 
  clrAriaLabelDropdownOpen="open something something" 
  clrAriaLabelDropdownClose="close something something..."
>
```

```typescript
// dropdown-trigger.ts
host: {
  ...
 [attr.aria-label]: 'setAriaLabel',
  ...
}

get setAriaLabel() {
  return this.active ? this.clrAriaLabelDropdownOpen : this.clrAriaLabelDropdownClose;
}

...
@Input('clrAriaLabelDropdownOpen') clrAriaLabelDropdownOpen: string = commonStrings.keys.ariaLabelDropdownOpen;
@Input('clrAriaLabelDropdownClose') clrAriaLabelDropdownClose: string = commonStrings.keys.ariaLabelDropdownClose;
```

And the default for all dropdown will be
```typescript
// common-strings.default.ts
export const commonStringsDefault: ClrCommonStrings = {
  ...
  ariaLabelDropdownOpen: 'open dropdown',
  ariaLabelDropdownClose: 'close dropdown',
}
```

Pretty much this is the how implementation.. without test and TypeScript notation :)

Reply:
For now, this fix is documentation and this is something we should backlog as a separate task if we prioritize it.